### PR TITLE
Buff captain skills

### PR DESCRIPTION
**Pull request by psykzz**
## About The Pull Request
Captain while still a shipside role is a strong role, they have many years in training and had to have shown skill in all aspects, in order to get that promotion to Captain.

## Why It's Good For The Game

More people playing captain.

## Changelog
:cl:
balance: captain got a buff in skills.
/:cl:

### CODE REVIEW

**Comment by Chaznoodles** [comment]
Synth 2.0

**Comment by psykzz** [comment]
except captain is a shipside role, so who cares.
If it gets to SD and captain wants to run around solo fine.

If that means people take captain at round start, i think its welcome change.

**Review by TiviPlus — COMMENTED** [review]
Looks ok I guess except for the surgery change, that feels like stepping on MO and CMOs toes too much

**Comment by Marmio64** [comment]
yeah, all of its great besides surgery, which should be made amateur like the other command roles. Just cause you're a captain in the navy doesnt mean you're now a expert in surgery, you aren't taught how to re attach limbs in officer school.

File: code/datums/skills.dm [hunk 1]
```diff
@@ -324,14 +324,24 @@ engineer, construction, leadership, medical, surgery, pilot, police, powerloader
 
 /datum/skills/captain
 	name = CAPTAIN
+	leadership = SKILL_LEAD_MASTER
+	police = SKILL_POLICE_MP
+	medical = SKILL_MEDICAL_COMPETENT
+	surgery = SKILL_SURGERY_PROFESSIONAL
```
Comment (Hughgent) [inline code/datums/skills.dm:330]:
```suggestion

```
Surgery skill bad

**Review by Hughgent — APPROVED** [review]

**Pull request merged**